src/components/FilmCardComponent/FilmCard.jsx

```diff
@@ -3,6 +3,7 @@ import { withRouter } from 'react-router';
 import PropTypes from 'prop-types';
 import styles from './FilmCard.module.css';
 import ApiClient from '../../api/apiClient';
+import { formatDate } from '../../utils/utils';
```

Comment:
Please check this path. Are you sure that this file exists?

Reply:
Of course, I've added it

Reply:
Yeah, I'm forgot to add this file to commit. I've updated.